src/repository.ts

```diff
@@ -213,6 +213,19 @@ class RepositorySourceControlManager {
 
     this.parentResourceGroups = updatedGroups;
 
+    const trackedFilesList = await this.repository.fileList();
+    const trackedFiles = new Set<string>();
+
+    for (const t of trackedFilesList) {
+      const pathParts = t.split('/');
+      let currentPath = this.repositoryRoot + '/';
+      for (const p of pathParts) {
+        currentPath += p;
+        trackedFiles.add(currentPath);
```

Comment:
It seems like you're doing it this way so that `trackedFiles` also contains tracked folders, is that right?

Reply:
Yes, `jj file list` doesn't return directories. Git extension also do this for modified decoration, we may want to consider that as well.